templates/pkgs/base.json

```diff
@@ -19,6 +19,7 @@
   },
   "scripts": {
     "build": "sl-scripts build",
+    "bundle": "sl-scripts bundle",
```

Comment:
What would be the difference between build and bundle?

Reply:
build = plain tsc, a plain transpilation (same as v5 does)
bundle = creates a single file